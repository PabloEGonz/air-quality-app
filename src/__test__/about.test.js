import { render, screen } from "@testing-library/react"
import About from "./about";
import '@testing-library/jest-dom';
import React from "react";

test('it renders correctly', () => {
  render(<About />);
  expect(screen.getByText(/Qualitative/)).toBeInTheDocument();
});