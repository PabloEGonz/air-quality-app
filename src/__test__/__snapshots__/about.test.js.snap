// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders options page correctly 1`] = `[Function]`;
